frontend/views/containers/dashboard/GroupMembersActivity.vue

```diff
@@ -34,72 +27,97 @@
               //-   :args='{  members: 3 }'
               //-   @click='openModal("todoSeeMembers")'
               //- ) {members} members
-              i18n(:args='{ ...LTags("strong"), streaks: "4 weeks" }')  have {strong_} streaks of more than {streaks}{_strong}
+              i18n(:args='{ ...LTags("strong") }') have {strong_} on-time payment streaks{_strong}
 
     .c-column
       i18n.is-title-3(tag='h2') Inactivity
       .has-text-1.c-para Members that haven’t logged in, missed their pledges or haven´t voted last proposals will appear here.
 
       ul.spacer
-        li.c-item-wrapper
+        li.c-item-wrapper(v-if='haventLoggedIn.length > 0')
           .c-item
             .icon-user.icon-round.has-background-general
             .c-item-copy
-              tooltip(direction='bottom-end')
-                i18n.link(
-                  :args='{  members: 5 }'
-                ) {members} members
-                template(slot='tooltip')
-                  div Rosalía
-                  div Ken M
-                  div Ines de Castro
-                  div Attila the Hun
-                  div Istralianda
-              i18n(:args='LTags("strong")')  haven´t {strong_} logged in past week {_strong}
-
-        li.c-item-wrapper
+              member-count-tooltip.c-member-count(:members='haventLoggedIn')
+              i18n(:args='{ ...LTags("strong"), daysCount: config.notLoggedInDays }') haven´t {strong_} logged in past {daysCount} days or more {_strong}
```

Comment:
Two concerns here:

1. In some languages, the placement of "5 member" might be different, and by breaking up the translation, you may be generating a confusing / wrong translation. Sentences should always be translated as complete sentences, and never broken up for this reason. Sometimes it is technically difficult to do this, as might be the case here, but it's not unprecedented. For example, look at how `Payments.vue` handles opening the income modal when "income details" is clicked (lines 20-26)
2. Please use ' instead of ´ for the apostrophe (same issue below on line 63)

Similar concerns for the lines below.

Reply:
2. has been resolved in the follow-up commit.

regarding 1., thanks for providing the reference to `Payments.vue`. it's actually more complicated case here than how things is handled in `payments.vue`, because the side effect we want to achieve is toggling the tooltip, and the way `Tooltip.vue` is built is it has to wrap around the whole element that triggers it. so I tried to figure out a way to inject `<tooltip></tooltip>` into the `i18n` component but didn't really work.
So I believe it requires building a new type of tooltip component that allows us to set a specific part of its content as the `trigger` element. And is it alright for me to work on this as a standalone issue (or a follow-up work of this issue) after this PR has been merged? would be easier to work with if it's handled as a separate concern. :)

hope it sounds good. Thanks,

Reply:
@SebinSong Yeah, if you'd like to work on it in a followup please open up an issue and self-assign and reference this PR in it :)